### DISCUSSION TITLE
Jetson Nano or Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by mlittle408** [roflcoopter/viseron#292] [Q&A] [answered]
I have a jetson nano running the deepstack server that I plan to use for object detection. Am I better off running viseron on the Nano or the spare Raspberry PI 4 8GB that I have?

**Comment by roflcoopter** [comment, marked as answer]
Thats a great question, the Nano would probably be more ideal, but the hardware acceleration seems wonky at best.
The RPi 4 also has hardware acceleration support through the `h264_v4l2m2m` decoder in FFmpeg
